src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int NUMBER_OF_COLORS = 9;
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(NUMBER_OF_COLORS);
+        String result = Color.values()[index].toString();
+        return result;
+    }

```

Comment:
```suggestion
    private static final int NUMBER_OF_COLORS = 9;
    private final Random random = new Random();

    public String getRandomColor() {
        int index = random.nextInt(NUMBER_OF_COLORS);
        return Color.values()[index].toString();
    }
```

Reply:
next time if you don't understand comment you can ask to explain them
do not just ignore comments